projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -296,7 +303,7 @@ public function remove_core_styles() {
 	public function add_styles_and_scripts() {
 		// WoA sites: If wpcom_admin_interface is set to wp-admin, load the wp-admin styles.
 		// These include only styles to enable the "My Sites" and "Reader" links that will be added.
-		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' || $this->use_dev_nav ) {
 			$css_file = $this->is_rtl ? 'masterbar-wp-admin-rtl.css' : 'masterbar-wp-admin.css';
 			wp_enqueue_style( 'a8c-wpcom-masterbar-overrides', $this->wpcom_static_url( '/wp-content/mu-plugins/admin-bar/masterbar-overrides/' . $css_file ), array(), JETPACK__VERSION );
```

Comment:
This is the external css file that we would update separately to adjust the padding/spacing on the W logo in this branch.

Reply:
D137303-code